packages/gatsby-cypress/README.md

```diff
@@ -37,3 +36,40 @@ Once imported, the following additional commands are available:
     .get(`#element-with-event-handler`)
     .click()
   ```
+
+## Running Cypress tests in Gatsby
+
+Add a new script in `package.json` to run the Cypress tests. A common name for this is `cy:open`.
+
+You also need to expose a `CYPRESS_SUPPORT` environment variable to enable [Gatsby test utilities](https://github.com/gatsbyjs/gatsby/blob/1fb376f84ee538bac79824cd119bef6a17c19b33/packages/gatsby/cache-dir/api-runner-browser.js#L9-L18). Place it in your test script in `package.json`, for example:
+
+```diff:title=package.json
+  "scripts": {
+    ... other scripts ...
++   "cy:open": "cypress open",
++   "test:dev": "CYPRESS_SUPPORT=y start-server-and-test develop http://localhost:8000 cy:open",
++   "test": "CYPRESS_SUPPORT=y npm run build && start-server-and-test serve http://localhost:9000 cy:open"
+  }
+```
+
+> **NOTE:** `test:dev` runs the site in development mode, which allows you to quickly fix and retest any issues. `test` is better suited for build systems like Circle CI, Travis CI, etc.
+
+## Writing Cypress tests for Gatsby pages
```

Comment:
Why not link the already existing doc page here?

Reply:
@LekoArts what's the already existing page? I was trying to add a minimal example so that someone on this README can get tests up and running without needing to look elsewhere.

Reply:
https://www.gatsbyjs.org/docs/end-to-end-testing/

Reply:
I didn't add an example back when I created this page as the Cypress docs are pretty good. However, I'll need to edit the CI docs on this page to switch to `build` and `serve` on CI (could do that next week)

Reply:
Let's keep example here for now and once https://www.gatsbyjs.org/docs/end-to-end-testing/ is updated with example we can swap this section with "See https://www.gatsbyjs.org/docs/end-to-end-testing/ for example usage" (or something along this lines)

Reply:
Yeah, I'll work on this today a bit and send in a PR when finished 👍 But shouldn't be a blocker